heron/config/src/yaml/conf/yarn/uploader.yaml

```diff
@@ -1,5 +1,5 @@
 # uploader class for transferring the topology jar/tar files to storage
-heron.class.uploader:                            com.twitter.heron.uploader.NullUploader
+heron.class.uploader:                            com.twitter.heron.uploader.localfs.LocalFileSystemUploader
```

Comment:
@billonahill Yes, the YARN scheduler skips the uploader step. The heron binaries are added to **classpath** by the launcher in this case.